### PR TITLE
how-to/monitor: update manual deploy monitor

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
This PR updates the dashboard file for manual deploy monitor.

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1965

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

**Comment by anotherrachel** [comment]
@lilin90 @liubo0127 PTAL, thanks!

File: dev/how-to/monitor/monitor-a-cluster.md [hunk 1]
```diff
@@ -283,7 +283,7 @@ To import a Grafana dashboard for the PD server, the TiKV server, and the TiDB s
 
     > **Note:**
     >
-    > For the TiKV, PD, and TiDB dashboards, use the `tikv_pull.json` file, the `pd.json` file, the `tidb.json` file respectively.
+    > For the TiKV, PD, and TiDB dashboards, the corresponding JSON files are `tikv_summary.json`, `tikv_details.json`,`tikv_trouble_shooting.json`, `pd.json`, `tidb.json`, and `tidb_summary.json`.
```
Comment (lilin90) [inline dev/how-to/monitor/monitor-a-cluster.md]:
```suggestion
    > For the TiKV, PD, and TiDB dashboards, the corresponding JSON files are `tikv_summary.json`, `tikv_details.json`, `tikv_trouble_shooting.json`, `pd.json`, `tidb.json`, and `tidb_summary.json`.
```

File: v3.0/how-to/monitor/monitor-a-cluster.md [hunk 2]
```diff
@@ -284,7 +284,7 @@ To import a Grafana dashboard for the PD server, the TiKV server, and the TiDB s
 
     > **Note:**
     >
-    > For the TiKV, PD, and TiDB dashboards, use the `tikv_pull.json` file, the `pd.json` file, the `tidb.json` file respectively.
+    > For the TiKV, PD, and TiDB dashboards, the corresponding JSON files are `tikv_summary.json`, `tikv_details.json`,`tikv_trouble_shooting.json`, `pd.json`, `tidb.json`, and `tidb_summary.json`.
```
Comment (lilin90) [inline v3.0/how-to/monitor/monitor-a-cluster.md]:
```suggestion
    > For the TiKV, PD, and TiDB dashboards, the corresponding JSON files are `tikv_summary.json`, `tikv_details.json`, `tikv_trouble_shooting.json`, `pd.json`, `tidb.json`, and `tidb_summary.json`.
```

File: dev/how-to/monitor/monitor-a-cluster.md (hunk 1 above)
Comment (anotherrachel) [reply dev/how-to/monitor/monitor-a-cluster.md]: sharp eyes👍

**Pull request merged**